bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v18/BitcoindV18RpcClientTest.scala

```diff
@@ -0,0 +1,62 @@
+package org.bitcoins.rpc.v18
+import org.bitcoins.rpc.client.common.BitcoindVersion
+import org.bitcoins.rpc.client.v18.BitcoindV18RpcClient
+import org.bitcoins.testkit.rpc.BitcoindRpcTestUtil
+import org.bitcoins.testkit.util.BitcoindRpcTest
+
+import scala.concurrent.Future
+
+class BitcoindV18RpcClientTest extends BitcoindRpcTest {
+  lazy val client: BitcoindV18RpcClient = {
+    new BitcoindV18RpcClient(BitcoindRpcTestUtil.v18Instance())
+  }
+
+  clientAccum.+=(client)
+
+  behavior of "BitcoindV18RpcClient"
+
+  it should "be able to start a V18 bitcoind instance" in {
+
+    client.start().map { _ =>
+      assert(client.version == BitcoindVersion.V18)
+    }
+
+  }
+
+  it should "return active rpc commands" in {
+    val startF = client.start()
+    startF.flatMap(_ =>
+      client.getNewAddress.flatMap(client.generateToAddress(50000, _)))
+    val rpcinfoF = startF.flatMap(_ => client.getRpcInfo())
+
+    rpcinfoF.map { result =>
+      assert(result.active_commands.length == 2)
+    }
+  }
+
+  it should "return a list of wallets" in {
+    val startF = client.start()
+    val listF = startF.flatMap(_ => client.listWalletDir())
+
+    listF.map { result =>
+      assert(
+        result.wallets.head.name.length >= 25 && result.wallets.head.name.length <= 34)
+      assert(result.wallets.nonEmpty)
```

Comment:
This assertion should come before the above so that there is a nicer error message if it isn't satisfied (the previous line will throw)